docs/projects/cn0363/index.rst

```diff
@@ -0,0 +1,285 @@
+
+-  :git-hdl:`CN0363_PMDZ HDL project source code <projects/cn0363>`
+-  :dokuwiki:`[Wiki] CN0540 HDL project documentation <resources/eval/user-guides/eval-cn0363-pmdz/reference_hdl>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac <library/axi_dmac>`
+     - :ref:`here <axi_dmac>`
+   * - AXI_GENERIC_ADC
+     - :git-hdl:`library/axi_generic_adc <library/axi_generic_adc>`
+     - ---   
+   * - AXI_HDMI_TX
+     - :git-hdl:`library/axi_hdmi_tx <library/axi_hdmi_tx>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_hdmi_tx>`
+   * - AXI_I2S_ADI
+     - :git-hdl:`library/axi_i2s_adi <library/axi_i2s_adi>`
+     - ---
+   * - AXI_SPDIF_TX
+     - :git-hdl:`library/axi_spdif_tx <library/axi_spdif_tx>`
+     - ---
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid <library/axi_sysid>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - CN0363_DMA_SEQUENCER
+     - :git-hdl:`library/cn0363/cn0363_dma_sequencer <library/cn0363/cn0363_dma_sequencer>`
+     - :dokuwiki:`[Wiki] <resources/fpga/peripherals/cn0363/sequencer?s[]=dma&s[]=sequencer>`
+   * - CN0363_PHASE_DATA_SYNC
+     - :git-hdl:`library/cn0363/cn0363_phase_data_sync <library/cn0363/cn0363_phase_data_sync>`
+     - :dokuwiki:`[Wiki] <resources/fpga/peripherals/cn0363/phase_data_sync>`
+   * - CORDIC_DEMOD
+     - :git-hdl:`library/cordic_demod <library/cordic_demod>`
+     - ---
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine <library/spi_engine/axi_spi_engine>`
+     - :ref:`here <spi_engine axi>`
+   * - SPI_ENGINE_EXECUTION
+     - :git-hdl:`library/spi_engine/spi_engine_execution <library/spi_engine/spi_engine_execution>`
+     - :ref:`here <spi_engine execution>`
+   * - SPI_ENGINE_INTERCONNECT
+     - :git-hdl:`library/spi_engine/spi_engine_interconnect <library/spi_engine/spi_engine_interconnect>`
+     - :ref:`here <spi_engine interconnect>`
+   * - SPI_ENGINE_OFFLOAD
+     - :git-hdl:`library/spi_engine/spi_engine_offload`
+     - :ref:`here <spi_engine offload>`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom <library/sysid_rom>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - UTIL_AXIS_RESIZE
+     - :git-hdl:`library/util_axis_resize`
+     - ---   
+   * - UTIL_I2C-MIXER
+     - :git-hdl:`library/util_i2c_mixer <library/util_i2c_mixer>`
+     - ---
+   * - UTIL_SIGMA_DELTA_SPI
+     - :git-hdl:`library/util_sigma_delta_spi <library/util_sigma_delta_spi>`
+     - :dokuwiki:`[Wiki] <resources/fpga/peripherals/util_sigma_delta_spi>`
+
+-  :ref:`SPI Engine Framework documentation <spi_engine>`
+
+Software related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :git-linux:`CN0363 Linux driver source code <analogdevicesinc/linux/blob/main/drivers/iio/adc/ad_adc.c>`
```

Comment:
```
- :git-linux:`CN0363 Linux driver source code <drivers/iio/adc/ad_adc.c>`
```
https://analogdevicesinc.github.io/doctools/docs_guidelines.html#git-role

Reply:
Done in V2.